quadrat/sass/blocks/_query.scss

```diff
@@ -0,0 +1,64 @@
+.is-style-quadrat-staggered-posts {
+	li {
+		display: grid;
+		grid-column-gap: 20px;
+		grid-template-columns: 1fr 1fr;
+		grid-template-areas: 
+			"post-image post-title"
+			"post-image post-excerpt"
+			"post-image post-author"
+			"post-image post-date"
+			"post-separator post-separator";
+		&:nth-child(even){
+			grid-template-areas: 
+				"post-title post-image"
+				"post-excerpt post-image"
+				"post-author post-image"
+				"post-date post-image"
+				"post-separator post-separator";
+		}
+		& div {
+			//this ignores extra divs added by the editor so grid behaves correctly
+			display: contents;
+		}
+	}
+	.wp-block-post-title,
+	.wp-block-post-excerpt,
+	.wp-block-post-featured-image,
+	.wp-block-separator,
+	.wp-block-post-date,
+	.wp-block-latest-posts__post-author,
+	.wp-block-latest-posts__post-date,
+	.wp-block-latest-posts__featured-image,
+	.wp-block-latest-posts__post-excerpt {
+		//undos the above fix
+		display: revert;
```

Comment:
It was either this or using :not on the rule that adds display: contents; I might revisit this and do it some other way, there's 3 levels of extra divs that the editor is adding...